components/raftstore/src/store/worker/pd.rs

```diff
@@ -1111,13 +1121,40 @@ where
                     let peer_stat = self
                         .region_peers
                         .entry(hb_task.region.get_id())
-                        .or_insert_with(PeerStat::default);
+                        .or_insert_with(|| PeerStat::new(hb_task.peer.get_id()));
+                    if let Some(peer_id) = peer_stat.peer_id {
+                        if peer_id != hb_task.peer.get_id() {
+                            error!(
```

Comment:
I haven't been able to reproduce the issue nor this specific code path locally.

Reply:
I came across it on the big cluster before, maybe try to reproduce on it? I think it better to find the root cause.

Reply:
I checked our big cluster metrics in last 3 months, it did appear that region written bytes occasionally reached hundreds of PBs. However I think it's a separate issue, because in this case `tikv_region_written_bytes_sum` is around 40ZiB, no where near the `u64` underflow result which is more than 1e10 ZiB.

Reply:
I think there is something fundamentally wrong with region written stats, because in normal cases its value is already very spiky.